File: ChatAppAngular/src/app/user/payload/message-request.ts
import { ChatFiles } from "../models/chat-files";

export class MessageRequest {
    message: string='';
    sender: string='';
    recipient: string='';
    conversationId: string='';
    operationType: string='';
	messageFiles: ChatFiles[] = [];
	files:File[]=[]
  
    constructor(
      message: string,
      sender: string,
      recipient: string,
      conversationId: string,
      operationType: string
    ) {
      this.message = message;
      this.sender = sender;
      this.recipient = recipient;
      this.conversationId = conversationId;
      this.operationType = operationType;
    }
  }
  

File: ChatAppBackend/src/main/java/com/gapshap/app/payload/MessageRequest.java
package com.gapshap.app.payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MessageRequest {

	private String message;

	private String sender;

	private String recipient;

	private String conversationId;

	private String operationType;
//	private  = new ArrayList<>(); 

}
